### PR TITLE
data: krishnanunnijs

### PR DESCRIPTION
**Pull request by krishnanunnijs**
## Changes proposed

Added new profie link

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3607"></a>

### CODE REVIEW

File: data/krishnanunnijs.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Krishnan Unni",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```
Comment (SaraJaoude) [inline data/krishnanunnijs.json:5]: Please update your bio with your information

**Review by Tejender1521 — CHANGES_REQUESTED** [review]
Please update bio with your information and use your github avatar url link.

**Review by krishnanunnijs — COMMENTED** [review]
updated bio

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**